mission-board/src/main/java/com/board/article/domain/Article.java

```diff
@@ -0,0 +1,39 @@
+package com.board.article.domain;
+
+import com.board.member.domain.Member;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Article {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "memberId")
+    private Member member;
+    @Column
+    private String title;
+    @Column
+    private String content;
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "memberId")
    private Member member;
    
    @Column
    private String title;
    
    @Column
    private String content;
```

nullable 속성도 추가해주면 좋을 것 같아요 ㅎㅎ

Reply:
@Nullable 속성 추가하였습니다!